docs/UserGuide.md

```diff
@@ -168,6 +168,15 @@ _Help window displayed after running the `help` command_
 
 <br>
 
+### Navigate command history: `↑` `↓`
+
+Navigate the command history. Pressing `↑` will replace the text in the command box with the previous command. Pressing `↓` will replace the text in the command box with the next command.
+
```

Comment:
I think it might be good to mention that we can only navigate through successful commands